doc/chapter-firststeps.asciidoc

```diff
@@ -24,127 +25,137 @@ usage: ./newsboat [-i <file>|-e] [-u <urlfile>] [-c <cachefile>] [-x <command> .
-searched for it. When you do a search from the list of feeds, all articles of
-all feeds will be searched. When you do a search from the article list of a
-feed, only the articles of the currently viewed feed are searched. When opening
-an article from a search result dialog, the search phrase is highlighted.
-
-The history of all your searches is saved to the filesystem, to the
+=== First UI Interaction
+
+The main UI of Newsboat consists of three views
+
+_Feed List View -> Article List View -> Article View_
+
+You can drill down those views by pressing `Enter` and move to the previous one by pressing
+`q`. Pressing `q` on the _Feed List View_ — or pressing `Q` from anywhere — closes Newsboat.
+
+You can also search an articles' title or content by pressing `/` on the _Feed List View_ or the _Article List View_.
+On the _Feed List View_ all articles of all feeds are taken into account.
+On the _Article List View_ the articles of the current feed are taken into account.
+When opening an article from a search result dialog, the search phrase is highlighted.
+
+****
+*Search history* The history of all your searches is saved to the filesystem, to the
 _history.search_ file (stored next to the _cache.db_ file). By default, the
-last 100 search phrases are stored, but this limited can be influenced through
-the <<history-limit,`history-limit`>> configuration variable. To disable search
-history saving, simply set the `history-limit` to `0`.
-
-Keys, as used in the <<bind-key,`bind-key`>> configuration command, use a special syntax.
-Lowercase keys, uppercase keys and special characters are written literally.
-The "Enter" key is written as `ENTER`, "Tab" key as `TAB`, and the "Esc" key is
-written as `ESC`. The function keys "F1" to "F12" are written as `F1` to `F12`.
-The "Space" key is written as `SPACE`. Key combinations with the "Ctrl" key,
-such as "Ctrl-R", are written as `^R`. Please be aware that all Ctrl-related
-key combinations need to be written in uppercase. The following identifiers for
-keys are supported:
-
-- `ENTER` (Enter key)
-- `BACKSPACE` (backspace key)
-- `LEFT` (left cursor)
-- `RIGHT` (right cursor)
-- `UP` (up cursor)
-- `DOWN` (down cursor)
-- `PPAGE` (page up cursor)
-- `NPAGE` (page down cursor)
-- `HOME` (cursor to beginning of list/article)
-- `END` (cursor to end of list/article)
-- `ESC` (Esc key)
-- `TAB` (Tab key)
-
-Newsboat also comes with user contributed content like scripts and color
-themes. The user contributed content can be found in
-`/usr/share/doc/newsboat/contrib/`. End users are encouraged to take a look as
-they may find something useful.
+last 100 search phrases are stored.
+
+You can influence how many search phrases are stored by configuring <<history-limit,`history-limit`>>.
+****
+
+*Feed List View*
+
+If you start Newsboat, it presents you with a list of feeds that you added previously.
+
+You can now
+
+* press `R` to download articles for all feeds.
```

Comment:
I just realized that "press" is redundant in all of those list items. I'm not sure whether moving it up reads better. I tend to like it better like it is now. Any opinions on that?

Reply:
I guess it's easier to grasp when skimming, so this is good.

Reply:
I agree, I like it the way it is now. The only thing that *could* be changed is adding a colon to the `You can now:` introduction and start every bullet point in capitals, because the sentences all end in full stops. But nothing major.

Reply:
My intention was to make every option look like the completion of the sentence, that's why I didn't use a colon here at first. But it works just the same with a colon and every list item being its own sentence :+1:  Switched to colon and uppercased first letter of list items!